converters/postgres.go

```diff
@@ -0,0 +1,31 @@
+package converters
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+type Pg struct{}
```

Comment:
Don't export Pg or MySQL struct{}, they get registered from the converters package and would only end up cluttering the godoc